### PR TITLE
refactor(std): migrate common surface model to std

### PR DESCRIPTION
**Pull request by doouding**
### Changed
Add basic surface model. And make the previous one extend the basic surface.

### CODE REVIEW

**Comment by doouding** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7694?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#7694** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7694?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7592** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7592?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @doouding and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,5 @@ next-env.d.ts
 
 # Cache
 .eslintcache
+
+__screenshots__/
```
Comment (doodlewind) [inline .gitignore:32]: How is this being used?

**Review by Saul-Mirone — APPROVED** [review]
LGTM

File: .gitignore (hunk 1 above)
Comment (doouding) [reply .gitignore:32]: It seems the vitest will generate screenshot during running.

**Review by doodlewind — APPROVED** [review]

**Pull request merged**